graph_shortest_path/routing.py

```diff
@@ -42,7 +42,14 @@ def find_vertex(self, value):
         @return {Vertex} the found Vertex
         """
         # !!!! IMPLEMENT ME
-        pass
+        pop_index = 0
+        for v in self.vertices:
+          if self.vertices.pop(pop_index) == value:
+            return self.vertices
+          else:
+            pop_index +=1
+        return None
+
```

Comment:
This doesn't make sense...if you're looping through self.vertices..why are you ONLY checking for the first element's value? 
and why return the entire self.vertices?